update/plan/update_230000.go

```diff
@@ -0,0 +1,269 @@
+					var lazySchedules []types.LazySchedule
+					for _, s := range vs.Schedules {
+						lazySchedules = append(lazySchedules, types.NewLazySchedule(s.GetCliff(), s.GetCliff()+secondsPerMonth, s.GetRatio()))
+					}
+
+					lazyVestingSchedule := types.NewLazyVestingSchedule(vs.GetDenom(), lazySchedules)
+					lazyVestingSchedules = append(lazyVestingSchedules, lazyVestingSchedule)
+				}
+			}
+
+			for _, lvs := range lazyVestingSchedules {
+				if !lvs.IsValid() {
+					panic("not valid schdule")
+				}
+			}
+
+			baseAccount := &auth.BaseAccount{
+				Address:       gvacc.GetAddress(),
+				PubKey:        gvacc.GetPubKey(),
+				Coins:         gvacc.GetCoins().Sort(),
+				AccountNumber: gvacc.GetAccountNumber(),
+				Sequence:      gvacc.GetSequence(),
+			}
+
+			baseVestingAcc := &auth.BaseVestingAccount{
+				BaseAccount:      baseAccount,
+				OriginalVesting:  gvacc.GetOriginalVesting(),
+				DelegatedFree:    gvacc.GetDelegatedFree(),
+				DelegatedVesting: gvacc.GetDelegatedVesting(),
+				EndTime:          gvacc.GetEndTime(),
+			}
+
+			lazyAccount := types.BaseLazyGradedVestingAccount{
+				BaseVestingAccount:   baseVestingAcc,
+				LazyVestingSchedules: lazyVestingSchedules,
+			}
+
+			accKeeper.SetAccount(ctx, lazyAccount)
+		}
+
+		return
+	})
+
+	// update oracle reward band param
+	oracleParams := oracleKeeper.GetParams(ctx)
+	oracleParams.OracleRewardBand = sdk.NewDecWithPrec(2, 2) // 2%
+	oracleKeeper.SetParams(ctx, oracleParams)
+}
+
+// preseed account does not have any other schedules
+func updatePreseedSchedules(gvacc types.GradedVestingAccount) []types.LazyVestingSchedule {
+
+	vestingSchedules := gvacc.GetVestingSchedules()
+	if len(vestingSchedules) != 1 || vestingSchedules[0].GetDenom() != assets.MicroLunaDenom {
+		panic("Invalid Preseed Account")
+	}
+
+	vestingSchedule := vestingSchedules[0]
+	if len(vestingSchedule.Schedules) != 4 {
+		panic("Invalid Preseed Account")
+	}
+
+	// strict preseed account check
+	for _, s := range vestingSchedule.Schedules {
+		if s.GetCliff() == 1558677600 && s.GetRatio().Equal(sdk.NewDecWithPrec(10, 2)) {
+			continue
+		} else if s.GetCliff() == 1561356000 && s.GetRatio().Equal(sdk.NewDecWithPrec(10, 2)) {
+			continue
+		} else if s.GetCliff() == 1563948000 && s.GetRatio().Equal(sdk.NewDecWithPrec(10, 2)) {
+			continue
+		} else if s.GetCliff() == 1587708000 && s.GetRatio().Equal(sdk.NewDecWithPrec(70, 2)) {
+			continue
+		} else {
+			panic("Invalid Pressed Account")
+		}
+	}
+
+	var lazyVestingSchedules []types.LazyVestingSchedule
+	var lazyVestingSchedule types.LazyVestingSchedule
+	var lazySchedules []types.LazySchedule
+
+	genesisTime := time.Unix(genesisUnixTime, 0)
+	lazySchedules = append(lazySchedules,
+		types.NewLazySchedule(
+			genesisTime.AddDate(0, 1, 0).Unix(), genesisTime.AddDate(0, 2, 0).Unix(), sdk.NewDecWithPrec(10, 2)),
+		types.NewLazySchedule(
+			genesisTime.AddDate(0, 2, 0).Unix(), genesisTime.AddDate(0, 11, 0).Unix(), sdk.NewDecWithPrec(27, 2)),
```

Comment:
This should be 12, not 11

Reply:
Think even distribution from month 12 to 17 should be fine

Reply:
According to preseed schedule should have 3% release during 2~10 month, 4% release during 11~12 month,  10% release during 13~16 month, and 15% release at 17 month.

I think it should 11 to fit 2~10 month schedule.